Rebuild-Black-Business/RBB-Website
# Add image to T&C Modal on allies page

# Describe your PR
Adds an image to the modal dialog on the allies page for T's & C's.  It also centers the button on that dialog.

Related to [trello card](https://trello.com/c/PIEBMMW0/105-add-image-to-allies-modal-gating)

## Pages/Interfaces that will change
`/allies` the first time you visit it (before you click to accept)

### Screenshots / video of changes
desktop:
![image](https://user-images.githubusercontent.com/1844496/84289518-91132980-ab10-11ea-985b-806347141aca.png)

mobile:
![image](https://user-images.githubusercontent.com/1844496/84289538-983a3780-ab10-11ea-911a-9abb5fca7cb7.png)


## Steps to test

1. clear your cache so that any record of you accepting terms and conditions before is gone (otherwise the modal won't show)
1. visit `/allies` by clicking "Allies" in nav
1. check it out on desktop/modal

### Additional questions
- [ ] This PR doesn't match the Figma designs exactly -  the desktop view should show the image _to the left_ of the copy/button, but I was having trouble lining that up, at least partly because...

- [ ] The figma design also calls for the T&C modal to have a checkbox and a button that just says "Continue" - I can do that if you'd like, but thought I'd leave it since it wasn't called out specifically.

- [ ] the figma design also calls for a pill-shaped button, which isn't in the current implementation
